src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/InProcessRequestHandler.vcxproj

```diff
@@ -72,7 +71,7 @@
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>$(PlatformToolsetVersion)</PlatformToolset>
-    <WholeProgramOptimization>true</WholeProgramOptimization>
+    <WholeProgramOptimization>false</WholeProgramOptimization>
```

Comment:
This optimization results in
```
LINK : fatal error LNK1000: Internal error during IMAGE::BuildImage [C:\Github\aspnetcore\src\Servers\IIS\AspNetCoreMod
uleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj]
```

Reply:
Please open an issue about restoring optimization